lib/sandbox/execute.js

```diff
@@ -72,23 +86,57 @@ module.exports = function (bridge, glob) {
             module: _module
         });
 
-        scope.exec(template, (err) => {
-            if (err) {
-                return bridge.dispatch('initialize', err);
-            }
+        for (const templateName of Object.keys(templates || {})) {
+            const template = templates[templateName],
+                // eslint-disable-next-line no-await-in-loop, no-loop-func
+                templateExecResult = await new Promise((resolve) => {
```

Comment:
Why not use `Promise.all`?

Reply:
Need to audit the perf here. If initializing all templates at once is significantly increasing the initialization time, it'll negatively affect single request execution scenarios.

In that case we can go for on-demand initialization